pl/firefox-ios.xliff

```diff
@@ -2863,7 +2863,7 @@
   <body>
    <trans-unit id="Bookmarks Menu">
     <source>Bookmarks Menu</source>
-    <target>Zakładki</target>
+    <target>Menu Zakładki</target>
```

Comment:
For the record, this is how it looks on Android:

![screenshot_20170228-184845](https://cloud.githubusercontent.com/assets/139229/23485059/1f33c2c4-feda-11e6-963b-c5b3120f3e7f.png)

I’ve removed “Menu” from the string in firefoxios.